src/test/java/net/snowflake/ingest/streaming/internal/datatypes/IcebergStructuredIT.java

```diff
@@ -58,29 +58,25 @@ public void testStructuredDataType() throws Exception {
     assertStructuredDataType("array(string)", null);
 
     /* Map with null key */
-    SFException ex =
-        Assertions.catchThrowableOfType(
-            SFException.class,
+    Assertions.assertThatThrownBy(
             () ->
                 assertMap(
                     "map(string, int)",
                     new HashMap<String, Integer>() {
                       {
                         put(null, 1);
                       }
-                    }));
-    Assertions.assertThat(ex)
-        .extracting(SFException::getVendorCode)
+                    }))
+        .isInstanceOf(SFException.class)
+        .extracting("vendorCode")
         .isEqualTo(ErrorCode.INVALID_FORMAT_ROW.getMessageCode());
 
     /* Unknown field */
-    ex =
-        Assertions.catchThrowableOfType(
-            SFException.class,
+    Assertions.assertThatThrownBy(
             () ->
-                assertStructuredDataType("object(a int, b string)", "{\"a\": 1, \"c\": \"test\"}"));
-    Assertions.assertThat(ex)
-        .extracting(SFException::getVendorCode)
+                assertStructuredDataType("object(a int, b string)", "{\"a\": 1, \"c\": \"test\"}"))
+        .isInstanceOf(SFException.class)
+        .extracting("vendorCode")
         .isEqualTo(ErrorCode.INVALID_FORMAT_ROW.getMessageCode());
```

Comment:
also validate the string messages in these tests, not just the error code.

Reply:
Added error message validation.